packages/mui-lab/src/internal/pickers/hooks/usePickerState.ts

```diff
@@ -43,6 +43,13 @@ export function usePickerState<TInput, TDateValue>(
 
   const utils = useUtils();
   const { isOpen, setIsOpen } = useOpenState(props);
+  const [initialDate, setInitialDate] = React.useState<TInput>(value);
+
+  React.useEffect(() => {
```

Comment:
I don't think you need this. This hook maintains its own state and therefor the `value` prop should be considered the default value, correct?

Reply:
We don't need to use `isOpen` as a conditional to set `initialDate` but we need to newly set the value of `initialDate` when `onAccept` is called. Otherwise, when you click "cancel", the value of date will always go back to the original value when you first loaded the component.